src/Breadcrumbs/Breadcrumb.php

```diff
@@ -117,8 +117,12 @@ class Breadcrumb
      */
     public function __construct($name, $type, array $metaData = [])
     {
-        if (!is_string($name) || $name === '') {
-            throw new InvalidArgumentException('The breadcrumb name must be a non-empty string.');
+        if (!is_string($name)) {
+            throw new InvalidArgumentException('The breadcrumb name must be a string.');
+        }
+
+        if ($name === '') {
+            $name = 'NULL name given';
```

Comment:
This is not true. `null !== ''`. In fact, this won't fix the issue for the person who was passing null, since `is_string(null) === false`.